torchci/lib/bot/labelBot.ts

```diff
@@ -0,0 +1,106 @@
+import { Probot } from "probot";
+
+function labelBot(app: Probot): void {
+  // labels start with non space char \\w and follow by list of labels separated by comma
+  const labelCommand = new RegExp(
+    "@pytorchbot\\s+label\\s+(\\w[\\w\\s.:,&-/]+)"
+  );
+
+  app.on("issue_comment.created", async (ctx) => {
+    const commentBody = ctx.payload.comment.body;
+    const owner = ctx.payload.repository.owner.login;
+    const repo = ctx.payload.repository.name;
+    const commentId = ctx.payload.comment.id;
+    const prNum = ctx.payload.issue.number;
+    // maximum specified in the doc
+    const labelsPerPage = 100;
+
+    async function reactOnComment(reaction: "+1" | "confused") {
+      await ctx.octokit.reactions.createForIssueComment({
+        comment_id: commentId,
+        content: reaction,
+        owner,
+        repo,
+      });
```

Comment:
This code looks almost identical to the one in mergeBot and other places. Perhaps we can create a base class or something and inherit from it?

Reply:
Done. Moved to torchci/lib/bot/botUtils.ts